src/care/LoopFuser.h

```diff
@@ -1254,6 +1255,7 @@ void LoopFuser<REGISTER_COUNT, XARGS...>::registerAction(const char * fileName,
 
 #define FUSIBLE_KERNEL_R_END return 0;}); } FUSIBLE_FLUSH_IF_NEEDED
 #define FUSIBLE_KERNEL_PHASE_R_END FUSIBLE_KERNEL_R_END
+#define FUSIBLE_KERNEL_PHASE_END FUSIBLE_KERNEL_R_END
```

Comment:
Should this have the FUSIBLE_FLUSH_IF_NEEDED or not?

Reply:
Also, I noticed that FUSIBLE_LOOP_STREAM_R doesn't have if (END > START), but FUSIBLE_LOOP_PHASE_R does.  Is that intentional?

Reply:
> Should this have the FUSIBLE_FLUSH_IF_NEEDED or not?

I think it is good as is (calling FUISBLE_FLUSH_IF_NEEDED through FUSIBLE_KERNEL_R_END)

Reply:
It does seem like the END > START check is worth doing for all the loop types at first glance. It's been a while since I did this, but I don't think I bumped into anything that required calling the infrastructure for what would be no-ops.

Reply:
How about the FUSIBLE_FLUSH_IF_NEEDED?  It is omitted in FUSIBLE_LOOP_PHASE_R_END